Assets/SEE/Game/HolisticMetrics/ActionHelpers/BoardAdder.cs

```diff
@@ -64,7 +69,12 @@ internal static void Init()
             boardPrefab = Resources.Load<GameObject>("Prefabs/HolisticMetrics/SceneComponents/MetricsBoard");
             positioningDone = false;
             gotPosition = false;
-            GameObject.Find("/DemoWorld/Plane").AddComponent<BoardAdder>();
+            GameObject floor = GameObject.Find(floorPath);
+            if (floor == null)
+            {
+                throw new System.Exception($"Could not find the floor in the scene. Please make sure the floor is named '{floor}'.");
```

Comment:
Should probably be `floorPath` instead, as `floor` is null here:
```suggestion
                throw new System.Exception($"Could not find the floor in the scene. Please make sure the floor is at '{floorPath}'.");
```